src/main/java/net/pms/remote/RemoteUtil.java

```diff
@@ -175,16 +175,16 @@ public static void sendLogo(HttpExchange t) throws IOException {
 		dump(in, os);
 	}
 
-	public static boolean directmime(String mime) {
+	public static boolean directmime(String mimeType) {
 		if (
-			mime != null &&
+			mimeType != null &&
 			(
-				mime.equals(MIME_MP4) ||
-				mime.equals(MIME_WEBM) ||
-				mime.equals(MIME_OGG) ||
-				mime.equals(MIME_MP3) ||
-				mime.equals(MIME_PNG) ||
-				mime.equals(MIME_JPG)
+				mimeType.equals(MIME_MP4) ||
+				mimeType.equals(MIME_WEBM) ||
```

Comment:
@Sami32 I was wrong, there is audio transcoding. It will play ogg files just fine as it is. I don't have any WEBP audio files so I can't test that.